hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -533,4 +536,36 @@ void maxGasIsPositive() {
             }
         }
     }
+
+    @Test
+    void maxGasForFoundryDeterministicDeployerIsAsExpected() {
+        final var oneByte = new byte[] {1};
+        // high bit of most significant byte is zero
```

Comment:
Isn't this comment related to the line above? If so, we can move it on top.

Reply:
removed the comment.  I copied from the unit test above this one.  I don't think that comment adds much clarity.